.github/workflows/test.yaml

```diff
@@ -43,12 +43,20 @@ jobs:
       - name: Prepare build
         run: yarn e2e:pre-prod
       
-      - name: Run tests
+      - name: Run admin user tests
+        continue-on-error: true
         run: |
-          yarn e2e:prod
+          GREP_TAGS=@adminUser TEST_USERNAME=admin yarn e2e:prod
```

Comment:
These two grep tags will select tests with the admin / user tags. All tests without any tags aren't run.

I had a look at the `not` flag to select all admin and not standard users... but that will exclude any tests tagged with both.

We might need to add the admin tag to all describes

Reply:
Correct, I only added the tags to a few specs because I was debugging up until now but I'll add the tags to the remaining tests asap. My thinking is to add both the adminUser and standardUser tags to all the tests except I wouldn't add the standardUser tag to tests that are specific to an admin such as the Rancher setup tests. Thoughts? https://github.com/rancher/dashboard/blob/f7a3b09bc116fa0e94358c98ed718e6a6f767d3b/cypress/e2e/tests/setup/rancher-setup.spec.ts#L4

Reply:
I went ahead and added the admin and standard user tags where applicable in this commit https://github.com/rancher/dashboard/pull/9029/commits/5dba5bcd78d79abcd6d81b8fb223b212e79e993. According to the [logs from that test run](https://github.com/rancher/dashboard/actions/runs/5294262015/jobs/9583405643), it looks like the standard user tests completed within 13 minutes which is not as concerning as I thought.